.docker/aiida-core-with-services/s6-assets/s6-rc.d/rabbitmq/run

```diff
@@ -2,5 +2,14 @@
 
 with-contenv
 
-foreground { s6-echo "Calling /etc/init/rabbitmq.sh" }
-rabbitmq-server
+foreground { s6-echo "Starting RMQ server and notifying back when the service is ready" }
+
+
+# For the container that includes the services, aiida-prepare.sh script is called as soon as the RabbitMQ startup script has
+# been launched, but it can take a while for the RMQ service to come up. If ``verdi presto`` is called straight away
+# it is possible it tries to connect to the service before that and it will configure the profile without a broker.
+# Here we use s6-notifyoncheck to do the polling healthy check of the readyness of RMQ service.
+#
+# -w 500: 500 ms between two invocations of ./data/check
+
+s6-notifyoncheck -w 500 rabbitmq-server
```

Comment:
```suggestion
# -w 1000: 1000 ms between two invocations of ./data/check

s6-notifyoncheck -w 1000 rabbitmq-server
```

Reply:
500 ms is anyway too short, in my console I saw 3 ~4 rmq not booting yet error until it continues. So 1s interval is more proper.